docs/docs/integrating-with-other-libraries.md

```diff
@@ -0,0 +1,205 @@
+---
+id: integrating-with-other-libraries
+title: Integrating with other libraries
+permalink: docs/integrating-with-other-libraries.html
+---
+
+## Using jQuery plugins from React
+
+React is unaware of changes made to the DOM outside of React. It determines updates based on its own internal representation, and if those updates are invalidated, React has no way to recover.
+
+This does not mean it is impossible or even necessarily difficult to combine React with other ways of affecting the DOM, you just have to be mindful of what each are doing.
+
+The easiest way to avoid conflicts is to prevent the React component from updating. This can be done explicitly by returning false from [`componentWillUpdate()`](https://facebook.github.io/react/docs/react-component.html#componentwillupdate), or by rendering elements that have no reason to change.
+
+```js
+class SomePlugin extends React.Component {
+  componentDidMount() {
+    this.$el.somePlugin();
+  }
+
+  componentWillUnmount() {
+    // cleanup
```

Comment:
Various plugin libs have their own setup/destroy approaches, but I think `$el.somePlugin("destroy")` is a typical and clear enough example that it should go in there.

Using `$` is absolutely clear, I'm just suggesting to do that as a separate step in `componentDidMount` so it's not kind of 'buried" in the ref callback.  It'll also help illustrate that the normal use for a callback ref is to save the reference to the real DOM node.